README.md

```diff
@@ -102,11 +102,17 @@ protocol:
   - git pull
 ```
 
-## Running inside `docker`
+## Running inside `docker` (for OSX users)
 
 You can run the tool inside docker, for example like this:
 
 ``` bash
 ./build-docker-image.sh
 ./check-protocols-in-docker.sh <PATH_TO_YOUR_SCRIPT>
 ```
+
+and for local development, you can run tests continuously
+
+``` bash
+./test-watch-in-docker.sh
+```
```

Comment:
Now there are two fairly different use-cases for the docker image:

- Run the tool as a user.
- Hack on the rust code of the tool.

I think that's fine, but I think this should be separated more in the README. Maybe a section about running the tool in docker and then another section `Contributing` that would include how to develop on a mac? If you want I can take a stab at that.

Reply:
updated